misc/flatpak/org.qutebrowser.qutebrowser.json

```diff
@@ -0,0 +1,197 @@
+{
+    "app-id": "org.qutebrowser.qutebrowser",
+    "branch": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "qutebrowser",
+    "finish-args": [
+        /* Allow direct rendering to use OpenGL and access to webcams */
+        "--device=all",
+        /* Allow to read all files from the home directory */
+        "--filesystem=home:ro",
+        /* Allow to save downloads */
+        "--filesystem=xdg-download",
+        /* Allow IPC access for performance reasons (X shared memory extension) */
+        "--share=ipc",
+        /* Allow network access */
+        "--share=network",
+        /* Allow to play sounds using pulseaudio */
+        "--socket=pulseaudio",
+        /* Allow to provide a graphical UI */
+        "--socket=wayland",
+        "--socket=x11"
+    ],
+    "rename-desktop-file": "qutebrowser.desktop",
+    "rename-appdata-file": "qutebrowser.appdata.xml",
+    "rename-icon": "qutebrowser",
+    "cleanup": [
+        "/include",
+        "/bin/sip",
+        "/lib/debug"
+    ],
+    "modules": [
+        {
+            "name": "PyQT5",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 configure.py --confirm-license --assume-shared --verbose --sip=/app/bin/sip --sip-incdir=/app/include/python3.5 --destdir /app/lib/python3.5/site-packages --bindir /app/bin --qml-plugindir=/app/lib/plugins/PyQt5 --no-sip-files --no-stubs --no-tools --no-qml-plugin",
```

Comment:
I wonder, would it be possible to use the prebuilt wheel on x86_64 but this everywhere else?